crypto/Cargo.toml

```diff
@@ -13,12 +13,15 @@ default = ["std"]
 std = ["ursa"]
 # Force static linking
 vendored = ["openssl-sys"]
-# Generate extern functions callable via FFI
-ffi = ["iroha_ffi"]
+## Replace structures and methods with FFI equivalents. Facilitates dynamic linkage
+#ffi = ["iroha_ffi/client"]
+
+# Expose FFI API for dynamic linking (Internal use only)
+ffi_api = ["std"]
```

Comment:
activated by `iroha_core`

Reply:
Again, name is a little confusing. Docstring isn't usually displayed for features.

Reply:
I was inspired by `mutable_api` feature name. It expands the API by exposing ffi functions

Reply:
`mutable_api` works, because you have the context of "this is API for the `data_model`, this is the API that allows you to mutate the objects". Here, the distinction between the `ffi` and the `ffi_api` is very murky. It won't stop me from approving, just something we should ponder before it comes to merging.